tooling/ast_fuzzer/src/compare/interpreted.rs

```diff
@@ -173,11 +173,22 @@ impl Comparable for ssa::interpreter::errors::InterpreterError {
             | (
                 ConstrainNeFailed { lhs: _lhs1, rhs: _rhs1, msg: msg1, .. },
                 ConstrainNeFailed { lhs: _lhs2, rhs: _rhs2, msg: msg2, .. },
+            )
+            | (
+                ConstrainNeFailed { lhs: _lhs1, rhs: _rhs1, msg: msg1, .. },
+                ConstrainEqFailed { lhs: _lhs2, rhs: _rhs2, msg: msg2, .. },
+            )
+            | (
+                ConstrainEqFailed { lhs: _lhs1, rhs: _rhs1, msg: msg1, .. },
+                ConstrainNeFailed { lhs: _lhs2, rhs: _rhs2, msg: msg2, .. },
             ) => {
```

Comment:
We could simplify this whole match to
```noir
(
    ConstrainEqFailed { msg: msg1, .. } | ConstrainNeFailed { msg: msg1, .. },
    ConstrainEqFailed { msg: msg2, .. } | ConstrainNeFailed { msg: msg2, .. },
)
```

Reply:
Yeah.. I originally left it to emphasise, but you're right it's silly.